src/network/protocol/messages/filterclear.zig

```diff
@@ -0,0 +1,54 @@
+const std = @import("std");
+const protocol = @import("../lib.zig");
+
+/// FilterClear represents the "filterclear" message
+///
+/// https://developer.bitcoin.org/reference/p2p_networking.html#filterclear
+pub const FilterClearMessage = struct {
+    // FilterClear message do not contain any payload, thus there is no field
+
+    pub inline fn name() *const [12]u8 {
+        return protocol.CommandNames.FILTERCLEAR ++ [_]u8{0} ** 5;
```

Comment:
`FILTERCLEAR` is already 11 char + final `\0`. No need to append anything to it in order to reach `*const [12]u8`